apps/judicial-system/api/src/app/modules/case/interceptors/case.transformer.spec.ts

```diff
@@ -176,4 +178,55 @@ describe('transformCase', () => {
       expect(res.isAppealGracePeriodExpired).toBe(true)
     })
   })
+
+  describe('isStatementDeadlineExpired', () => {
+    it('should be false if the case has not been appealed', () => {
+      // Arrange
+      const theCase = { appealState: undefined } as Case
+
+      // Act
+      const res = transformCase(theCase)
+
+      // Assert
+      expect(res.isStatementDeadlineExpired).toBe(false)
+    })
+
+    it('should be false when more than one day has passed since the case was appealed', () => {
+      // Arrange
```

Comment:
Shouldn't it be false "when lass than one day has passed"? and then in line 214 "when more than one day"

Reply:
Yes! Thanks